src/store-tree/index.spec.ls

```diff
@@ -19,7 +19,6 @@ describe 'StoreTree' ->
   describe '$debug' ->
 
     test-cases '' [
-      -> @tree = create-tree name: __, email: __
```

Comment:
This example isn't possible anymore as the placeholders get normalized. I didn't want to convert from `test-cases` to a single `beforeEach` in this PR as it would have caused a much bigger diff. I can clean these in up after this get merged.